src/libs/Navigation/OnyxTabNavigator.tsx

```diff
@@ -1,12 +1,24 @@
 import {createMaterialTopTabNavigator, MaterialTopTabNavigationEventMap} from '@react-navigation/material-top-tabs';
 import {EventMapCore, NavigationState, ScreenListeners} from '@react-navigation/native';
 import React from 'react';
+import {Platform} from 'react-native';
 import {withOnyx} from 'react-native-onyx';
 import {OnyxEntry} from 'react-native-onyx/lib/types';
 import Tab from '@userActions/Tab';
 import ONYXKEYS from '@src/ONYXKEYS';
 import ChildrenProps from '@src/types/utils/ChildrenProps';
 
+const TabNavigatorAnimationEnabled = () => {
+    switch (Platform.OS) {
```

Comment:
since this is stable, do we have to call it with each render?

Reply:
Sure thing! Done ✅